system/Files/File.php

```diff
@@ -128,17 +126,9 @@ public function guessExtension(): string
 	 */
 	public function getMimeType(): string
 	{
-		if (function_exists('finfo_file'))
-		{
-			$finfo = finfo_open(FILEINFO_MIME_TYPE);
-			$mimeType = finfo_file($finfo, $this->getRealPath());
-			finfo_close($finfo);
-		}
-		else
-		{
-			$mimeType = mime_content_type($this->getRealPath());
-		}
-
+		$finfo = finfo_open(FILEINFO_MIME_TYPE);
```

Comment:
Why remove the check here? As an extension, it's possible that the file_info functions won't be available, though they are more secure than the `mime_content_type` function, which is why they're used by default. 

Your commit message says that `mime_content_type` is deprecated, but I don't see that [in the manual](https://secure.php.net/manual/en/function.mime-content-type.php). Where'd you get that info?